src/sidebar/templates/group-list.html

```diff
@@ -1,22 +1,32 @@
 <div class="pull-right"
      dropdown
      keyboard-nav>
+  <!-- Show a drop down menu if showGroupsMenu is true. -->
   <div class="dropdown-toggle"
         dropdown-toggle
-        data-toggle="dropdown"
-        role="button"
-        title="Change the selected group">
+        data-toggle="dropdown" ng-if="vm.showGroupsMenu()">
     <img class="group-list-label__icon group-list-label__icon--organization"
          ng-src="{{ vm.focusedIcon() }}"
          alt="{{ vm.orgName(vm.groups.focused().id)}}"
          ng-if="vm.focusedIcon()">
          <i class="group-list-label__icon h-icon-{{ vm.focusedIconClass() }}"
              ng-if="!vm.focusedIcon()"></i><!-- nospace
-    !--><span class="group-list-label__label">{{vm.groups.focused().name}}</span><!-- nospace
-      !--><i class="h-icon-arrow-drop-down"></i>
+    !--><span class="group-list-label__label"><a class="group-list-label__toggle">{{vm.groups.focused().name}}</a><!-- nospace
```

Comment:
@dwolfe The span tags are in a different place than before-is that intentional/ok?

Reply:
Yep, that's intentional. I literally just changed the `<h2>` to a `<span>` and looked to make sure it appeared correctly, and it did. Wouldn't hurt to have confirmation of that if you want to take a look, but my hope was that it would be just as simple to change it back with the new PR.